unime/src/routes/+layout.svelte

```diff
@@ -32,16 +31,19 @@
     dispatch({ type: '[App] Get state' });
   });
 
-  let showDevMode = false;
-
+  let expandDevMenu = true;
```

Comment:
Suggestion: if you feel the need to rename `show` to `expand`, I suggest being consistent in naming and call it `expandedDevMenu` (or something alike) since "expand" describes a process and would rather fit to a function and not a boolean state.

Reply:
Because the dev menu is still shown, in both cases the one is just collapsed. I will rename to expanded